apps/examples/security_test/seclink/sl_crypto_test.c

```diff
@@ -209,6 +209,44 @@ START_TEST_F(aes_ctr)
 }
 END_TEST_F
 
+START_TEST_F(gcm_aes)
+{
+	hal_data aes_key = HAL_DATA_INITIALIZER;
+	hal_data enc = HAL_DATA_INITIALIZER;
+	hal_data dec = HAL_DATA_INITIALIZER;
+	HAL_INIT_GCM_PARAM(param);
+	unsigned char aad[16] = {0,};
+	unsigned char tag[16] = {0,};
+	aes_key.data = g_key_128;
+	aes_key.data_len = 16;
+	param.cipher = HAL_GCM_AES;
+	param.iv = (unsigned char *)g_iv;
+	param.iv_len = 16;
+	param.aad = aad;
+	param.aad_len = 16;
+	param.tag = tag;
+	param.tag_len = 16;
+
+	enc.data = g_plaintext;
+	enc.data_len = 16;
+	dec.data = g_ciphertext;
+	dec.data_len = 16;
+
+	ST_EXPECT_EQ(SECLINK_OK, sl_set_key(g_hnd, HAL_KEY_AES_128, ST_AES_ENC_KEY_IDX, &aes_key, NULL));
+	ST_EXPECT_EQ(SECLINK_OK, sl_gcm_encrypt(g_hnd, &dec, &param, ST_AES_ENC_KEY_IDX, &enc));
```

Comment:
Hello @ZhenBei-Sin 
I tested with this tesetcode, but I got error while calling `sl_gcm_encrypt`.

```
[ RUN        ] gcm_aes repeat: 1
keymgr request cmd(2580) (0) key(32) isempty(1)
hal_result_e : 0
hal_result_e : 255

security level: 0
===========================================================
Assertion details
===========================================================
print_assert_detail: Assertion Failed CPU0 at file: src/seclink/seclink.c line 130 task: sl_test pid: 20
print_assert_detail: Assert location (PC) : 0x0e179d83
check_assert_location: Code asserted in normal thread!
```

The location if assert is below.
https://github.com/Samsung/TizenRT/blob/master/framework/src/seclink/seclink.c#L129

When sl_gcm_encrypt called, it returns 255, not the defined value in res_table.
Would you please kindly let me know what I set wrong?

Reply:
Hi @jylee9613 

In this PR was missing the secure bl32.bin. (This image contains the SE API which support AES GCM mode).
![image](https://github.com/user-attachments/assets/6b0290ee-5718-460e-a3b4-02635fc68d94)

Verified SUCCESS gcm_aes.
![image](https://github.com/user-attachments/assets/9c17fff7-9b63-43b4-b929-1268cf1548ae)

Thank You.

Reply:
OMG...Thank you for checking:)
I confirmed AES GCM runs properly in AIDual.
I'll also check with TP1x, and let you know the result.

Thank you